ethereum-types/Cargo.toml

```diff
@@ -1,24 +1,24 @@
 [package]
 name = "ethereum-types"
-version = "0.4.0"
+version = "0.5.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 license = "MIT"
 homepage = "https://github.com/paritytech/primitives"
 description = "Ethereum types"
 
 [dependencies]
-crunchy = "0.1"
+crunchy = { version = "0.2.1", default-features = false }
```

Comment:
`version = "0.2"…`

Reply:
There is a problem. `crunchy` `2.0` and `crunchy` `2.1` are totally different in respect to their default features. So I would keep that one to make our use case explicit.